tests/test_formatter.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+from pydocstringformatter.formatting import FORMATTERS
+from pydocstringformatter.formatting.base import Formatter
+
+
+@pytest.mark.parametrize(
+    "formatter,expected", zip(FORMATTERS, ["beginning-quotes", "closing-quotes"])
+)
+def test_name(formatter: Formatter, expected: str) -> None:
```

Comment:
This might be hard to maintain at some point.

What about:
```python
formatter_names: Set[str] = {}
for formatter in FORMATTERS:
    assert formatter.name, "Each formatter should have a name set."
    assert formatter.name not in formatter_names, "Each formatter should have an unique name."
    formatter_names.add(formatter.name)
```

We could even separate the `name` property and this test into a separate PR to keep a clean commit history.

Reply:
Right, now that the name isn't generated there's no need to make sure that someone check that the "name" of the new formatter make sense.

Reply:
👍 

No, but an exclusivity check makes sense I think. Wouldn't hurt in any case.

Reply:
Done

Reply:
Opened up #18 and changed the branch we want to merge in here :)